app/routes/document-history.js

```diff
@@ -0,0 +1,84 @@
+module.exports = {
+  method: 'GET',
+  path: '/document-history',
+  options: {
+    handler: (request, h) => {
+      const DocumentID = request.query.DocumentID
+
+      // Fetch from API
+      const document = {
```

Comment:
Could these be moved out into a constants module? Just thinking it might make things easier to read when we're hooking into the APIs.

Reply:
Sorted